src/components/MAS/Mas.tsx

```diff
@@ -0,0 +1,137 @@
+/* Import Dependencies */
+import { useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import classNames from 'classnames';
+import { Tabs, TabList, Tab, TabPanel } from 'react-tabs';
+import { Container, Row, Col } from 'react-bootstrap';
+
+/* Import Store */
+import { useAppSelector, useAppDispatch } from 'app/Hooks';
+import { getMachineAnnotationService, setMachineAnnotationService } from 'redux/MAS/MASSlice';
+
+/* Import Utilities */
+import KeycloakService from 'keycloak/Keycloak';
+
+/* Import Components */
+import Header from 'components/Header/Header';
+import TitleBar from 'components/general/IDCard/titleBar/TitleBar';
+import IDCard from 'components/general/IDCard/IDCard';
+import MASServiceTab from './components/MASServiceTab';
+import MASTargetDigitalObjectFiltersTab from './components/MASTargetDigitalObjectFiltersTab';
+import MASDependenciesTab from './components/MASDependenciesTab';
+
+/* Import API */
+import GetMAS from 'api/mas/GetMAS';
+
+
+const MAS = () => {
+    /* Hooks */
+    const dispatch = useAppDispatch();
+    const params = useParams();
+    const navigate = useNavigate();
+
+    /* Base variables */
+    const MAS = useAppSelector(getMachineAnnotationService);
+
+    /* OnLoad: fetch Source System */
```

Comment:
I think this should be fetch Mapping

Reply:
Yeah should be Machine Annotation Service, copying stuff with comments and forgetting to edit them :D